popSimManu.tex

```diff
@@ -659,42 +661,43 @@ \section*{Methods}
 %\subsection*{Generating the PopSim resource}
 % RNG: No need to empty subsection.
 
-\subsection*{Model QC procedures}
+\subsection*{Model quality control}
 As a consortium we have agreed to a standardized procedure for model inclusion
 into \stdpopsim that allows for rigorous quality control. Imagine Developer A
 wants to introduce a new model into \stdpopsim. Developer A implements the
 demographic model for the relevant organism along with clear documentation
-of the model parameters and populations. This model is submitted as a Pull Request,
+of the model parameters and populations. This model is submitted as a ``pull request'',
 where it is evaluated by a reviewer and then included as `preliminary',
 but is not linked to the online documentation nor the command line interface.
-Inclusion of a preliminary model triggers a QC issue, and a second developer,
+Inclusion of a preliminary model triggers a quality control (QC) issue, and a second developer,
```

Comment:
I would make this active (since it's not automatic). (e.g "Developer A submits a quality control issue...")